MobileCenterUpdates/MobileCenterUpdatesTests/MSUpdateUtilTests.m

```diff
@@ -0,0 +1,39 @@
+//
```

Comment:
If you don't have any implementation of this class, let's remove it. It can come later with actual test.